Use django-admin instead of django-admin.py

toxinu opened the pull request:
Django 3.0 deprecated the `django-admin.py` command and Django 4.0 removed it in favor of `django-admin`.

https://docs.djangoproject.com/en/4.1/releases/4.0/

openedx-webhooks commented:
Thanks for the pull request, @toxinu!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.




<!-- comment:external_pr -->
<!-- jenkins ok to test -->
<!-- data: eyJkcmFmdCI6IGZhbHNlfQ== -->

pomegranited commented:
@toxinu This change is working fine! I'm not sure about whether the formatting changes are welcome (unsure about the lint tests that might be run for this repo?), but we can let upstream decide on that.

@Carlos-Muniz  since you are the CC most acquainted with the language translation tools, would you be able to help us get this change merged? Since `django-admin.py` was deprecated in Django 3, but not removed until Django 4, and [edx-platform is using Django 3](https://github.com/openedx/edx-platform/blob/master/requirements/common_constraints.txt#L21), there's no huge rush here, but it's a low-risk change to apply.

:+1:

- [x] I tested this by installing this branch on my (LX) devstack, and running the `i18n_tool dummy` and `i18n_tool generate` commands.
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] <strike>Includes documentation</strike> N/A

toxinu commented:
@pomegranited Thanks for your review. I didn't want to commit the formatting changes (I just had `black` running automatically). So I have updated the changes and rewritten the git history without the formatting changes.

pomegranited commented:
Thanks for getting these tests running @mphilbrick211 and @e0d !
Who's the best person/team to ping to get this reviewed and merged?

mphilbrick211 commented:
Hi @kashifch! Would you mind taking a look at this for review/merge? I believe you are the owner of this repo. Thank you!

Carlos-Muniz reviewed (approved):
LGTM
Thanks for catching the deprecation and updating it here!

pomegranited commented:
Thank you @Carlos-Muniz !
Hi @kashifch , can you help us get this merged? CC @mphilbrick211

mphilbrick211 commented:
Hi @kashifch - friendly follow-up on getting this merged :)

mphilbrick211 commented:
Hi @kashifch - friendly ping on this :)

pomegranited commented:
Thank you for keeping the communication going here, @mphilbrick211 :)

@Carlos-Muniz Would you be willing to add this repo as part of your core contributor merge rights, given your experience and efforts on internationalization and translations? I'm happy to nominate you, and/or define a "i18n CC repos group".  CC @sarina

sarina commented:
@pomegranited - I think an i18n CC repos group makes sense. Unfortunately, Carlos has moved on from his position at tCRIL. It would not be a bad idea to try to identify others who could at least assist with issues in these repos.

sarina commented:
@pomegranited @toxinu - what's the risk of merging this, if @kashifch is unavailable to do it?

sarina commented:
Actually - @kashifch I understand this change and will be merging it tomorrow unless you disagree. Thanks!

openedx-webhooks commented:
@toxinu 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/i18n-tools&entry.867055334=https://github.com/openedx/i18n-tools/pull/122&entry.1484655318=https://github.com/toxinu&entry.752974735=2022-12-22+13:53&entry.1917517419=2023-02-09+17:08&entry.2133058324=Yes) so we can improve your experience in the future.

<!-- comment:end_survey -->

toxinu commented:
Thanks, everybody for helping and merging this pull request. 😃 🎊
